Task 3/script.js

```diff
@@ -12,3 +12,71 @@ būti stilizuota su CSS ir būti responsive;
 -------------------------------------------------------------------------- */
 
 const ENDPOINT = 'https://api.github.com/users';
+
+//Function to get users list.
+async function getList(link) {
+  try {
+    const response = await fetch(link);
+    if (response.ok) {
+      return await response.json();
+    }
+  }
+  catch (error) {
+    console.log(error);
+  }
+}
+
+//Function to manage output.
+function showList(dataArray) {
```

Comment:
Funkcija per ilga. Jokia funkcija neturetu buti tokio ilgio. To galima butu isvengti CSS koda iskeliant i CSS faila.